libraries/AP_DDS/AP_DDS_Client.cpp

```diff
@@ -473,6 +520,9 @@ bool AP_DDS_Client::init()
 
     // Register topic callbacks
     uxr_set_topic_callback(&session, AP_DDS_Client::on_topic, &count);
+    
+    // ROS-2 Service : Register service request callbacks
+    uxr_set_request_callback(&session, AP_DDS_Client::on_request, 0);
```

Comment:
What does the last parameter of zero mean? Can you make it a constexpr variable?